Skeleton.md

```diff
@@ -0,0 +1,34 @@
+# Skeleton
+
+Skeleton is a dead simple boilerplate to kickstart any responsive project.
+
+## Getting Started
+
+Here is a simple HTML template which includes the latest compiled and minified CSS for the Skeleton library.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Skeleton Template</title>
+    <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.1.1/normalize.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css">
+  </head>
+  <body>
+    <h1>Hello World</h1>
+    <!-- Add all HTML Code here -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  </body>
+</html>
+```
+
+We have used a CDN in this example, but you can checkout other ways of installing Skeleton [here](https://github.com/dhg/Skeleton#getting-started).
+
+## Learning Resources
+
+- The official documentation for Skeleton is available [here](http://getskeleton.com/#grid).
```

Comment:
- :point_right: [**`Official Documentation`**](http://getskeleton.com/#grid) of Skeleton

```md
- :point_right: [**`Official Documentation`**](http://getskeleton.com/#grid) of Skeleton
```